packages/peregrine/lib/talons/AccountMenu/useAccountMenuItems.js

```diff
@@ -38,7 +38,7 @@ export const useAccountMenuItems = props => {
         {
             name: 'Saved Payments',
             id: 'accountMenu.savedPaymentsLink',
-            url: ''
+            url: '/saved-payments'
```

Comment:
Eventually we may be able to conditionally remove this route from the bundle.

See #2718